### PR TITLE
Fixing a few obsolete v1 references.

### PR DESCRIPTION
**Pull request by eranmes**

### CODE REVIEW

File: rfc6962-bis.xml [hunk 1]
```diff
@@ -1103,7 +1103,7 @@ messages.
         <t>
           Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be
 verified by constructing the Merkle Tree Hash corresponding to a retrieved
-STH. All leaves MUST be v1 or v2. However, a compliant v1 client MUST NOT construe an
+STH. All leaves MUST be v1 or v2. However, a compliant v2 client MUST NOT construe an
```
Comment (robstradling) [inline rfc6962-bis.xml]: "All leaves MUST be v1 or v2".  Does that make sense?  Or do we need to require that (in order to conform to this version of the specification) all leaves MUST be v2?  (i.e. each log is either entirely v1 or entirely v2).

Comment (eranmes) [inline rfc6962-bis.xml]: You're right, it doesn't make a lot of sense - v2 logs should be entirely  v2.

Comment (eranmes) [inline rfc6962-bis.xml]: There are further problems with this sentence because LogEntryType doesn't exist anymore. Looking into re-writing it.

**Comment by eranmes** [comment]
Done, PTAL.

File: rfc6962-bis.xml [hunk 2]
```diff
@@ -1101,13 +1101,7 @@ messages.
...
         </t>
         <t>
-          Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be
-verified by constructing the Merkle Tree Hash corresponding to a retrieved
-STH. All leaves MUST be v1 or v2. However, a compliant v1 client MUST NOT construe an
-unrecognized LogEntryType value as an error. This means it
-may be unable to parse some entries, but note that each client can inspect the
-entries it does recognize as well as verify the integrity of the data by
-treating unrecognized leaves as opaque input to the tree.
+          Note that this message is not signed -- the <spanx style="verb">entries</spanx> data can be verified by constructing the Merkle Tree Hash corresponding to a retrieved STH. All leaves MUST be v2. However, a compliant v2 client MUST NOT construe an unrecognized TransItem value as an error. This means it may be unable to parse some entries, but note that each client can inspect the entries it does recognize as well as verify the integrity of the data by treating unrecognized leaves as opaque input to the tree.
```
Comment (robstradling) [inline rfc6962-bis.xml]: "unrecognized TransItem value".  It's specifically the type of the TransItem that may be unrecognized.

I suggest: s/TransItem value/TransItem type/

Comment (eranmes) [inline rfc6962-bis.xml]: Done.

**Pull request merged**